nntrainer/tensor/blas_interface.cpp

```diff
@@ -246,6 +250,19 @@ void sgemv(CBLAS_ORDER order, CBLAS_TRANSPOSE TransA, const unsigned int M,
 #ifdef USE_BLAS
 #ifdef BLAS_NUM_THREADS
   openblas_set_num_threads(BLAS_NUM_THREADS);
+#endif
+
+#ifdef USE_NEON
+  if (TransA == CblasNoTrans) {
+    if (incX == 1 && incY == 1 && (N % 16 == 0 || N % 8 == 0 || N % 4 == 0)) {
```

Comment:
The (N % 16) and (N % 8) cases are included in (N % 4)

Reply:
Yes. but if 16 case is checked first and if it is done, then it returns without go further.